domain/src/main/java/com/sitionix/bffssox/client/AuthClient.java

```diff
@@ -0,0 +1,8 @@
+package com.sitionix.bffssox.client;
+
+import com.sitionix.bffssox.domain.User;
+
+public interface AuthClient {
+
+    User execute(final User user);
```

Comment:
whats doing this method and this interface

Reply:
i rename this method to createUser